source/Ubuntu-B.ufo/data/com.github.fonttools.ttx/T_S_I__0.ttx

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<ttFont sfntVersion="\x00\x01\x00\x00" ttLibVersion="3.0">
+<ttFont sfntVersion="\x00\x01\x00\x00" ttLibVersion="3.21">
```

Comment:
The `ttLibVersion` nump is diff noise, and ideally should be filtered out from the actual commit.

Reply:
I disagree. It’s important to track the fonttools version used to dump a ttx file as new attributes or elements may be added over time to the format, and parsers may take that into account

Reply:
Yes, in a *separate commit* that clearly states the origin of the change.  ie. that the change is fully understood and not diff noise that confuses the true purpose of the improvement being proposed.